### ISSUE TITLE
Reading videos on different pages

### ISSUE THREAD

**Issue by celfred** [OpenBoard-org/OpenBoard#84]
Using OpenBoard (OB) 1.3.5 on Ubuntu 14.04.
I had an issue with my video files and I read http://openboard.ch/forum/viewtopic.php?f=7&p=948&sid=b9ec90af87b41849a4f5028f28f01e41#p948
I tried the doc3.ubz mentionned in the Forum post and it worked successfully on my machine. But here's the result of my investigations : 
1. Import doc3.ubz
2. Close/Open OB > No problem reading the files (one, then the other, multiple stops/re-read...)
3. Documents mode : duplicate doc3
4. Redo steps 1 and 2 > Perfect.
5. Cut 1 video > Add a new page to doc3 > Paste video
6. Don't close OB! You can play/stop a video on either page : No problem.
7. Close / Open OB
8. Go to your copied file (with 2 pages and 1 video on each).
9. Play video on page 1, then stop : No problem.
10. Go to page 2 : Play video : NO IMAGE but sound is OK. You can't stop/play any longer...
11. Redo steps 7 through 10 by starting to read video on page 2. Then, video on page 1 has NO IMAGE :(

Launching OB from the console shows no error so I don't know the reason of this bug, but that's how it is.
It's a little hard to explain. I hope I was clear enough and that it'll help in some ways :)

PS : I tested with other videos and faced the same issue when separating my videos on multiple pages.

**Comment by freeheel3r** [comment]
So if I understand you correctly, discounting all the steps that work fine, the problem is that videos that were cut and pasted from one page to another stop working after closing then reopening OB?

**Comment by celfred** [comment]
In short, yes ;)
BUT : I have also tested putting videos directly from the library (dragging and dropping it) and the same behavior occurs : after closing/reopening OB, the first video you play (either 1 or 2) works fine, but the other one becomes unusable (unless you close/reopen OB and start with the other video).
Sorry if I was a little 'messy' in my explanations. I tried and explained every step so you can reproduce the bug on your side.

**Comment by freeheel3r** [comment]
Ha. I just tried to reproduce this on Windows and it crashed when I cut the video.
I'll check it out on Linux when I have the time. Maybe the two problems are related.

**Comment by celfred** [comment]
Ok. You're very welcome. Thank YOU for your consideration and your work on the project.

**Comment by celfred** [comment]
As a follow-up : 
2 days ago, I put my 2 videos on 1 page so it worked. I tried several times opening/closing OB with no problem.
Yesterday, at work, but on the same computer (a laptop) : the videos couldn't be played ! No image, but sound OK for none of them. I had to create another document and put my videos back from the library. It worked.
Today, the videos on this new document still work. But the first one seems definitely broken :(
Another test of mine : DocA : 1 page with 1 working video / DocB : new blank document. Drag'n'drop the docA page on docB : the video is unreadable on DocB and OB says 'unknown resource' (even after stopping/relaunching). So maybe this is a new issue : "Drag'n'dropping a page including a video doesn't work". I can create this as a new issue if someone can confirm (and if @freeheel3r thinks this has no link with the current issue (#84).
